mix_common.exs

```diff
@@ -11,13 +11,14 @@
 defmodule Antikythera.MixCommon do
   def common_project_settings() do
     [
-      elixir:            "~> 1.9",
-      elixirc_options:   [warnings_as_errors: true],
-      build_path:        build_path(),
-      build_embedded:    Mix.env() == :prod,
-      test_coverage:     [tool: ExCoveralls],
-      preferred_cli_env: [coveralls: :test, "coveralls.detail": :test, "coveralls.html": :test, "antikythera_local.upgrade_compatibility_test": :test],
-      xref:              [exclude: [EEx, EEx.Engine]], # Suppress undefined application warnings
+      elixir:                "~> 1.9",
+      elixirc_options:       [warnings_as_errors: true],
+      build_path:            build_path(),
+      build_embedded:        Mix.env() == :prod,
+      test_coverage:         [tool: ExCoveralls],
+      preferred_cli_env:     [coveralls: :test, "coveralls.detail": :test, "coveralls.html": :test, "antikythera_local.upgrade_compatibility_test": :test],
+      xref:                  [exclude: [EEx, EEx.Engine]], # Suppress undefined application warnings
+      consolidate_protocols: false, # Avoid inclusion of consolidated protocol information in the core PLT file
```

Comment:
You should write a comment that our release build doesn't use consolidate protocols.

If I look at this comment without prior knowledge, you disable consolidate protocols because you don't want to include them to the core PLT file although the disabling degrades performance in release. However, you know our release doesn't use consolidate protocols, so there is no performance degradation in release.

Reply:
Wrote the comment at 48f1713.